apidoc/Titanium/UI/Android/Android.yml

```diff
@@ -797,6 +797,20 @@ properties:
     permission: read-only
     since: "5.2.0"
 
+  - name: TABS_STYLE_DEFAULT
+    summary: The default TabGroup style that is an implementation similar to the (deprecated in Android) ActionBar.Tab implementation.
```

Comment:
I don't think we should mention the word "deprecated" here.
How about we just say this implements tabs above the content, but below the title bar?